doc_build_aliases.sh

```diff
@@ -145,6 +145,8 @@ alias docbldim='$GIT_HOME/docs/build_docs --doc $GIT_HOME/observability-docs/doc
 alias docbldfdg='$GIT_HOME/docs/build_docs --doc $GIT_HOME/observability-docs/docs/en/fleet-developer/index.asciidoc --chunk 2'
 
 # APM
+alias docbldapm='$GIT_HOME/docs/build_docs --doc $GIT_HOME/apm-server/docs/integrations-index.asciidoc --chunk 2'
+
```

Comment:
Do you want to change the `chunk` level in the aliases below to match what's in conf.yaml?